CODEOWNERS

```diff
@@ -9,6 +9,13 @@
 # will be requested to review.
 # *.js    @octocat @github/js
 
+# !!!! IMPORTANT !!!!
+# If a component is commented-out with a current contributor/team member
+# it means that the person still "owns" the component but does not want
+# to be automatically added as a reviewer to all PRs touching that component.
```

Comment:
I would expect an OWNER to be reviewing all incoming PRs against their components.
This comment is not likely to be visible to people opening PRs.

Reply:
This comment is to prevent people from removing commented-out owners. 

We are currently discussing the issue of listed owners reviewing every (possibly trivial) change to their component. If you would like to make that discussion public, perhaps we can open an Issue and debate it there?